sql/src/main/java/io/crate/expression/symbol/format/SymbolPrinter.java

```diff
@@ -187,7 +192,10 @@ private void printAnyOperator(Function function, SymbolPrinterContext context) {
             args.get(0).accept(this, context);
 
             // print operator
-            String operatorName = anyOperatorName(function.info().ident().name());
+            boolean ignoreCase = LikeOperators.extractIgnoreCase(function);
+            String functionName = function.info().ident().name();
+            String operatorName = caseSensitiveOperatorName(anyOperatorName(functionName), ignoreCase);
```

Comment:
Is this still required now that we have separate ilike operator functions?

Reply:
No, good eye!, thank you